recipes-kernel/kernel-modules/kernel-module-imx-gpu-viv_6.4.3.p4.6+fslc.bb

```diff
@@ -10,7 +10,8 @@ LIC_FILES_CHKSUM = "file://COPYING;md5=12f884d2ae1ff87c09e5b7ccc2c4ca7e"
 PV .= "+git${SRCPV}"
 
 SRCREV = "1adf982c5954f2f87919c6874279386f82e898a6"
-SRC_URI = "git://github.com/Freescale/kernel-module-imx-gpu-viv.git;protocol=https;branch=master"
+SRC_URI = "git://github.com/Freescale/kernel-module-imx-gpu-viv.git;protocol=https;branch=master \
+           file://0001-Change-return-type-of-_QuerySignal-from-gctBOOL-gceS.patch"
```

Comment:
Please send a PR to the repo so we avoid adding patch files.

Reply:
done